dbsnp-importer/src/main/java/uk/ac/ebi/eva/dbsnpimporter/jobs/steps/processors/RenormalizationProcessor.java

```diff
@@ -115,37 +126,101 @@ private Variant renormalize(IVariant variant) {
         } else if (variant.getAlternate().isEmpty()) {
             renormalizedReference = renormalizeAllele(variant.getReference());
         } else {
-            throw new AssertionError("Can not re-normalize: not a standard indel: " + variant);
+            throw new AssertionError("Can not re-normalize due to non-standard INDEL: " + variant);
         }
-        Variant renormalized = copyVariant(variant, renormalizedAlternate, renormalizedReference, renormalizedStart,
-                                           renormalizedEnd);
+        Variant renormalized = renormalizeVariant(variant, renormalizedAlternate, renormalizedReference, renormalizedStart,
```

Comment:
`createNormalizedVariant` would make more clear that the result is a new one. Same for similar methods.